### ISSUE TITLE
Dropdown attr aria-expanded missing because undefined value

### ISSUE THREAD

**Issue by cetincakiroglu** [primefaces/primeng#14832]
https://github.com/orgs/primefaces/discussions/1209

**Comment by TaneliTuomola** [comment]
May be duplicate of https://github.com/primefaces/primeng/issues/12590
TreeSelect and Calendar missing aria-expanded also: https://github.com/primefaces/primeng/issues/14358
